src/scores/continuous/flip_flop_impl.py

```diff
@@ -162,7 +162,7 @@ def flip_flop_index(
     if not selections and isinstance(data, xr.DataArray):
         result = _flip_flop_index(data, sampling_dim, is_angular=is_angular)
     else:
-        result = xr.Dataset()
+        result = xr.Dataset()  # type: ignore
```

Comment:
I'm concerned this ignore is too broad.
I would suggest specifying what is to be ignored, I assume `[assignment]`?

Reply:
I've raised a new issue to circle back and check this in all cases.